debian/changelog

```diff
@@ -1,3 +1,10 @@
+cloud-init (22.2-0ubuntu1~22.04.1) UNRELEASED; urgency=medium
```

Comment:
Note we can't reuse an already released version 22.2-0ubuntu1~22.04.1 in debian/changelog, so we'd want this to be
```suggestion
cloud-init (22.2-0ubuntu1~22.04.2) UNRELEASED; urgency=medium
```